szperacz/logger.py

```diff
@@ -1,36 +1,42 @@
 
 
 class Logger:
-    global log_level
-    log_level = 0
-    level = log_level
+    log_level = None
     DEBUG = '[DEBUG]'
     INFO = '[INFO]'
     ERROR = '[ERROR]'
+    LOG_LEVEL_DEBUG = 0
+    LOG_LEVEL_INFO = 1
+    LOG_LEVEL_ERROR = 2
 
-    error = open("error_log.txt", mode='w')
+    def __init__(self, level):
+        self.log_level = level
 
     def set_log_level(self):
-        pass
+        log_levels = [0, 1, 2]
```

Comment:
Use defined constants instead of arbitrary numbers:

```
def set_log_level(self, log_level):

  log_levels = [LOG_LEVEL_DEBUG, LOG_LEVEL_INFO, LOG_LEVEL_ERROR]

  if log_level not in log_levels:
    raise Exception(...)
  else:
    self.log_level = log_level 
```